### PR TITLE
Default target cluster

### PR DESCRIPTION
**Pull request by GordonSmith**
## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [ ] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by GordonSmith** [comment]
Is dependent on:  https://github.com/hpcc-systems/Visualization/pull/3254

File: packages/comms/src/services/wsStore.ts [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+            }
+            throw e;
+        });
+    }
+
+    FetchAll(request: WsStore.FetchAllRequest): Promise<WsStore.FetchAllResponse> {
+        return this._connection.send("FetchAll", request);
+    }
+
+    FetchKeyMD(request: WsStore.FetchKeyMDRequest): Promise<WsStore.FetchKeyMDResponse> {
```
Comment (jbrundage) [inline packages/comms/src/services/wsStore.ts]: Are these methods wrappers for existing services?

File: packages/comms/src/services/wsStore.ts [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+        return this._connection.send("FetchKeyMD", request);
+    }
+
+    ListKeys(request: WsStore.ListKeysRequest): Promise<WsStore.ListKeysResponse> {
```
Comment (jbrundage) [inline packages/comms/src/services/wsStore.ts]: Are these methods wrappers for existing services?

File: packages/comms/src/services/wsStore.ts [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+        return this._connection.send("ListKeys", request);
+    }
+
+    ListNamespaces(request: WsStore.ListNamespacesRequest): Promise<WsStore.ListNamespacesResponse> {
```
Comment (jbrundage) [inline packages/comms/src/services/wsStore.ts]: Are these methods wrappers for existing services?

Comment (GordonSmith) [reply packages/comms/src/services/wsStore.ts]: Yes all the files in the "services" folder are generated from existing services (well the request/response structures are).

**Pull request merged**